modules/deployment-scripts/aks-run-command/README.md

```diff
@@ -26,100 +31,12 @@ An Azure CLI Deployment Script that allows you to run a command on a Kubernetes
 
 ## Examples
 
-### Running a simple command
-
-```bicep
-module runCmd 'br/public:deployment-scripts/aks-run-command:1.0.1' = {
-  name: 'kubectlGetPods'
-  params: {
-    aksName: aksName
-    location: location
-    commands: [
-      'kubectl get pods'
-    ]
-  }
-}
-```
-
-### Using an existing managed identity
-
-When working with an existing managed identity that has the correct RBAC, you can opt out of new RBAC assignments and set the initial delay to zero.
-
-```bicep
-module runCmd 'br/public:deployment-scripts/aks-run-command:1.0.1' = {
-  name: 'kubectlGetNodes'
-  params: {
-    useExistingManagedIdentity: true
-    initialScriptDelay: '0'
-    managedIdentityName: managedIdentityName
-    rbacRolesNeeded:[]
-    aksName: aksName
-    location: location
-    commands: [
-      'kubectl get nodes'
-    ]
-  }
-}
-```
-
-### Sending multiple sequenced commands
-
-The module will sequence each command to run after the previous completes. There is a time impact for the DeploymentScript resource to create, therefore use new command items with consideration.
-
-```bicep
-var contributor='b24988ac-6180-42a0-ab88-20f7382dd24c'
-var rbacWriter='a7ffa36f-339b-4b5c-8bdf-e2c188b2c0eb'
-
-module runCmd 'br/public:deployment-scripts/aks-run-command:1.0.1' = {
-  name: 'kubectlRunNginx'
-  params: {
-    aksName: aksName
-    location: location
-    rbacRolesNeeded:[
-      contributor
-      rbacWriter
-    ]
-    commands: [
-      'kubectl run nginx --image=nginx'
-      'kubectl get pods'
-    ]
-  }
-}
-```
+### Example 1
```

Comment:
The Readme Examples have been lost

Reply:
@Gordonby , I can not figure out what happened here... "brm generate" will not produce new examples... 

I assume these should not be manually created?